src/image_copy/main.go

```diff
@@ -67,21 +70,68 @@ func getTags(dstTag string, updateMajor bool) []string {
 	return tags
 }
 
+func verifyImage(imageTag, platform, check string) {
+	fmt.Printf("verifying binaries in '%s' from '%s' for platform '%s' are '%s'\n", binFolder, imageTag, platform, check)
+
+	cmd := exec.Command(
+		"docker",
+		"run",
+		"--platform", platform,
+		"--user=root",
+		"--rm",
+		"--entrypoint=sh",
+		imageTag,
+		"-c", `
+apk add -U file > /dev/null
```

Comment:
You can use `text.Template` to clean this up

Reply:
@tdeebswihart what issue/advantage do you see here? I think string interpolation is fine for this use case. Since the quotes are easy to get right, escaping isn't an issue and gives you helpful syntax highlighting to see where the variables are.

Reply:
I don't, it's just a nit